files/image_config/watchdog-control/watchdog-control.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=watchdog control service
+After=rc-local.service
```

Comment:
It is probably make more sense to put this service 'After swss.service'

Reply:
I could go either way. Just depends on how early we feel comfortable disabling it, and whether or not we need to disable it early to prevent unnecessary watchdogs during normal reboot.

Reply:
We only use this service to recover fast/warm reboot so far. After swss is fine in that sense. We could also use this to protect against not able to initiate critical services. I agree that after swss could be an issue for cold reboot on wimpy platforms.